scripts/uploader.rb

```diff
@@ -192,7 +192,8 @@ def object_size(hash)
 
   # Does the whole upload process
   def upload(file, url, token)
-    puts "Uploading file #{file}"
+    filesize ='%.2f' % (File.size(file).to_f / 2**20)
```

Comment:
This should be named `file_size`. Also this text formatting type is not used anywhere else, so you should use a different text formatting type.  RubySetupSystem/SymbolUploader.rb has already print in MiB, you should copy the approach here.

Reply:
I change the variable name and format output text lile in RubySetupSystem/SymbolUploader.rb